dns/dnscrypt-proxy/src/opnsense/service/templates/OPNsense/Dnscryptproxy/dnscrypt-proxy.toml

```diff
@@ -40,6 +40,12 @@ doh_servers = true
 doh_servers = false
 {%   endif %}
 
+{%   if helpers.exists('OPNsense.dnscryptproxy.general.odoh_servers') and OPNsense.dnscryptproxy.general.odoh_servers == '1' %}
+odoh_servers = true
```

Comment:
How does this work… does it expect you to give ODoH servers explicitly or will it use a predefined pool? Main concern is users switching this on and expecting it to work when worst case it’s incompatible with their server selection.

Reply:
According to the documentation:

> Once one or more of the servers (whose name starts with odoh-) have been configured in the servers_list setting, the intermediate relays to reach them have to be configured.

So the odoh target servers will be pulled from the official DNSCrypt list but the relays have to be entered manually.

I could add a note to the menu comment like this: "Note: If checked you must provide a relay server!"